lib/shared.bash

```diff
@@ -60,6 +60,10 @@ function cache_restore_skip() {
   export BUILDKITE_PLUGIN_CACHE_HIT="false"
 }
 
+function cache_skip_s3_fetch() {
+  echo "Skip S3 fetch as cached archive already exists locally"
```

Comment:
lib/shared.bash  is shared between "tar/rsyns/s3/gcs" backends, hence messaging should be some kind generic here.